simapp/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	cosmossdk.io/depinject v1.0.0-alpha.3
 	cosmossdk.io/math v1.0.0-beta.4
 	cosmossdk.io/tools/rosetta v0.2.0
-	github.com/cosmos/cosmos-sdk v0.47.0-alpha2
+	github.com/cosmos/cosmos-sdk v0.47.0-rc2
```

Comment:
Changing this so it shows the correct version when going to `http://127.0.0.1:1317/cosmos/base/tendermint/v1beta1/node_info` from simapp built in the v0.47 branch.